src/styles/index.scss

```diff
@@ -1,3 +1,226 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  margin: 0;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header__nav-list {
+  margin: 0;
+  display: flex;
+  padding: 0;
+  list-style: none;
+}
+
+.header__nav-item {
+  align-items: center;
+  text-transform: uppercase;
+  margin-right: 20px;
+}
```

Comment:
Checklist item: 'All blocks should be the same width and height'. The '.header__nav-item' does not have a specified height, which could lead to inconsistency if the content changes.